### PR TITLE
Utility

### PR DESCRIPTION
**Pull request by Fleaman777**
New class: add test data in the data base.

### CODE REVIEW

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+class AddTestData {
+
+    public static Connection connection;
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: I think, you need `final` here, don't you?

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    private final static String SQL_ACCOUNTS = "INSERT INTO accounts (id_user, amount, is_active)" + " VALUES (?,?,?)";
+    private final static String SQL_CARDS = "INSERT INTO cards (id_user, id_account, is_active)" + " VALUES (?,?,?)";
+    private final static String SQL_ID_USER = "SELECT * FROM USERS WHERE login = ?";
+    private final static String SQL_ID_ACCOUNT = "SELECT * FROM ACCOUNTS WHERE id_user = ?";
+    private final static String SQL_ID_ROLE_USER = "SELECT * FROM USERS WHERE id_role = ?";
+
+    private static final int DATA_COUNT = 5;
+
+    private static void createData() {
+        USERS.add("zorita,1234,1,Zoy");
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: You may use smth like that (for better redability):
```
Users.addAll(
       Collections.asList(
                          "zorita,1234,1,Zoy",
                          "chaim,qwer,2,Chester",
                          ...
                          )
);
```

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        CARDS.add(true);
+        CARDS.add(true);
+        CARDS.add(false);
+        CARDS.add(true);
+        CARDS.add(true);
+    }
+
+    private static void generationData() throws SQLException, NamingException {
+        connection = ConnectionPool.getConnection();
+        int rand = (int) (Math.random() * USERS.size()) + 0;
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: Why do you need to add 0?

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+        String[] userArray;
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: No need to split declaring and assignment here

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        String nameUser = userArray[4];
+
+        PreparedStatement psUsers = connection.prepareStatement(SQL_USERS);
+        psUsers.setString(1, loginUser);
+        psUsers.setString(2, passwordUser);
+        psUsers.setInt(3, idRoleUser);
+        psUsers.setString(4, nameUser);
+        psUsers.executeUpdate();
+
+        String[] accountArray;
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: No need to split declaring and assignment here

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    public static boolean addTestData() {
+        try {
+            try {
+                if(checkedRoleAdmin()){
+                    return true;
+                }
+
+                for (int i = 0; i < DATA_COUNT; i++) {
```
Comment (Meisssshu) [inline src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: Do we really need this empty loop?

Comment (Fleaman777) [reply src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: This is my mistake, there should be a method: generationData()

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java (hunk 3 above)
Comment (Fleaman777) [reply src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: Yes, here it is not needed.

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java (hunk 2 above)
Comment (Fleaman777) [reply src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: Ok, I used Arrays.asList()

File: src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java (hunk 1 above)
Comment (Fleaman777) [reply src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java]: If I use the "final", I will need to immediately, when declaring a variable, handle the exception

File: src/main/java/com/epam/lab/paymentsystem/controller/UserController.java [hunk 7]
```diff
@@ -3,41 +3,38 @@
...
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
-
+@Controller
 public class UserController {
-
     private static final String REGISTRATION_PAGE = "registration";
+    private static final String ROOT = "redirect:/";
```
Comment (nikandtos) [inline src/main/java/com/epam/lab/paymentsystem/controller/UserController.java]: Separate redirect and mapping.

Comment (nikandtos) [inline src/main/java/com/epam/lab/paymentsystem/controller/UserController.java]: separate redirect and mapping

**Pull request merged**